whisper/whisperv6/peer_test.go

```diff
@@ -164,6 +164,11 @@ func initialize(t *testing.T) {
 			},
 		}
 
+		err = node.server.Start()
```

Comment:
Why is this necessary? We are getting an error further down `server already running`, which looks like might be triggered by this change, as I haven't seen this fail on go-ethereum/master

Reply:
dunno what happened here, but the code was different before:

```
diff --git a/whisper/whisperv5/peer_test.go b/whisper/whisperv5/peer_test.go
index bae2adb6f..cc9b05862 100644
--- a/whisper/whisperv5/peer_test.go
+++ b/whisper/whisperv5/peer_test.go
@@ -156,7 +156,7 @@ func initialize(t *testing.T) {
 
                err = node.server.Start()
                if err != nil {
-                       t.Fatalf("failed to start server %d.", i)
+                       t.Skipf("failed to start server %d (port may be taken, skipping since there is no handler in test for this, should be ported to simulation framework): error is %v", i, err)
                }
 
                nodes[i] = &node
diff --git a/whisper/whisperv6/peer_test.go b/whisper/whisperv6/peer_test.go
index 599a479be..b6a2f48c5 100644
--- a/whisper/whisperv6/peer_test.go
+++ b/whisper/whisperv6/peer_test.go
@@ -164,6 +164,11 @@ func initialize(t *testing.T) {
                        },
                }
 
+               err = node.server.Start()
+               if err != nil {
+                       t.Skipf("failed to start server %d (port may be taken, skipping since there is no handler in test for this, should be ported to simulation framework): error is %v", i, err)
+               }
+
                nodes[i] = &node
        }
 
@@ -174,7 +179,7 @@ func initialize(t *testing.T) {
        // we need to wait until the first node actually starts
        err = nodes[0].server.Start()
        if err != nil {
-               t.Fatalf("failed to start the fisrt server.")
+               t.Fatal("failed to start the first server: ", err)
        }
 }
```